### PR TITLE
add cases to test backup image with different cluster_size

### PR DESCRIPTION
**Pull request by xutian**
ID: 1808263, 1808264

Signed-off-by: Xu Tian <xutian@redhat.com>

### CODE REVIEW

**Comment by aliang123** [comment]
Hi, three scenarios are needed, and base(default cluster_size)
   1. inc_cluster_size > base_default_cluster_size
   2. inc_cluster_size < base_default_cluster_size
   3. inc_cluster_size = base_default_cluster_size

**Comment by xutian** [comment]
JOB ID     : 9e190559321dab03c178821fe3697540af439211
JOB LOG    : /root/avocado/job-results/job-2020-02-27T17.07-9e19055/job.log
 (1/4) Host_RHEL.m8.u1.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.1.0.x86_64.io-github-autotest-qemu.blockdev_inc_backup_cluster.inc_cluster_size.size_512.q35: PASS (100.70 s)
 (2/4) Host_RHEL.m8.u1.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.1.0.x86_64.io-github-autotest-qemu.blockdev_inc_backup_cluster.inc_cluster_size.size_4K.q35: PASS (101.54 s)
 (3/4) Host_RHEL.m8.u1.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.1.0.x86_64.io-github-autotest-qemu.blockdev_inc_backup_cluster.inc_cluster_size.size_2M.q35: PASS (102.48 s)
 (4/4) Host_RHEL.m8.u1.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.1.0.x86_64.io-github-autotest-qemu.blockdev_inc_backup_cluster.inc_cluster_size.q35: PASS (102.90 s)
RESULTS    : PASS 4 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0
JOB TIME   : 409.26 s
JOB HTML   : /root/avocado/job-results/job-2020-02-27T17.07-9e19055/results.html

File: qemu/tests/cfg/blockdev_inc_backup_cluster_test.cfg [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    image_name_inc = inc
+    source_images = "data"
+    rebase_mode = unsafe
+    only Linux
+    qemu_force_use_drive_expression = no
+    variants:
+        - inc_cluster_size:
+            variants:
+                - size_512:
+                    image_cluster_size_inc = 4096
```
Comment (aliang123) [inline qemu/tests/cfg/blockdev_inc_backup_cluster_test.cfg]: - size_512:
     image_cluser_size_inc = 512

Comment (xutian) [reply qemu/tests/cfg/blockdev_inc_backup_cluster_test.cfg]: fixed, thanks.

**Comment by aliang123** [comment]
(1/4) Host_RHEL.m8.u2.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.2.0.x86_64.io-github-autotest-qemu.blockdev_inc_backup_cluster.inc_cluster_size.size_512.q35: PASS (114.76 s)
 (2/4) Host_RHEL.m8.u2.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.2.0.x86_64.io-github-autotest-qemu.blockdev_inc_backup_cluster.inc_cluster_size.size_4K.q35: PASS (123.49 s)
 (3/4) Host_RHEL.m8.u2.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.2.0.x86_64.io-github-autotest-qemu.blockdev_inc_backup_cluster.inc_cluster_size.size_2M.q35: PASS (116.56 s)
 (4/4) Host_RHEL.m8.u2.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.2.0.x86_64.io-github-autotest-qemu.blockdev_inc_backup_cluster.inc_cluster_size.q35: PASS (115.88 s)

Acked-by: Aihua Liang<aliang@redhat.com>

**Comment by aliang123** [comment]
@xutian, please open an auto bug and add bug id. After created it, please ping me to check, thanks

**Comment by ngu-niny** [comment]
Test PASS:

'3-Host_RHEL.m8.u2.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.2.0.ppc64le.io-github-autotest-qemu.blockdev_inc_backup_cluster.inc_cluster_size.size_512' Finshed 2020-02-29 03:39:16 2020-02-29 03:46:45 PASS 448.592038
'4-Host_RHEL.m8.u2.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.2.0.ppc64le.io-github-autotest-qemu.blockdev_inc_backup_cluster.inc_cluster_size.size_4K' Finshed 2020-02-29 03:46:45 2020-02-29 03:50:38 PASS 233.22251
'5-Host_RHEL.m8.u2.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.2.0.ppc64le.io-github-autotest-qemu.blockdev_inc_backup_cluster.inc_cluster_size.size_2M' Finshed 2020-02-29 03:50:39 2020-02-29 03:54:37 PASS 238.566343
'6-Host_RHEL.m8.u2.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.2.0.ppc64le.io-github-autotest-qemu.blockdev_inc_backup_cluster.inc_cluster_size' Finshed 2020-02-29 03:54:37 2020-02-29 03:58:42 PASS 244.478639

Acked-by: Nini Gu <ngu@redhat.com>

**Pull request merged**